src/components/shared/Errors/GenericError.module.scss

```diff
@@ -0,0 +1,15 @@
+.asLink {
+  display: block;
+  background: none !important;
+  border: none;
+  padding: 0 !important;
+  /*optional*/
+  text-decoration: underline;
+  cursor: pointer;
+}
+button.asLink {
+  font-size: 1rem;
+}
+.errorLink {
+  color: #d4351c;
```

Comment:
Again, may be able to resolve this with the new classes in design system. https://github.com/wmcadigital/wmn-design-system/blob/master/src/wmnds/components/button/_button.scss : Line 240